python/core/qgsdiagramrendererv2.sip

```diff
@@ -94,6 +94,9 @@ class QgsDiagramSettings
 
     void readXML( const QDomElement& elem, const QgsVectorLayer* layer );
     void writeXML( QDomElement& rendererElem, QDomDocument& doc, const QgsVectorLayer* layer ) const;
+
+    //! @note caller is responsible for deletion of QStandardItems
+    QList< QStandardItem * > legendItems() const;
```

Comment:
I'm not 100% sure, but possibly these should be marked /Factory/ so that python will clean the created items when they go out of scope